generated/machinelearningservices/microsoft.machinelearningservices/2021-03-01-preview/types.md

```diff
@@ -525,6 +538,7 @@ This property will always be returned as null. AuthKey values must be retrieved
 ### SynapseSpark
 #### Properties
 * **computeType**: 'SynapseSpark' (Required): The type of compute
+* **properties**: [SynapseSparkPoolPropertiesProperties](#synapsesparkpoolpropertiesproperties): AKS properties
```

Comment:
This property was defined via [multiple inheritance](https://github.com/Azure/azure-rest-api-specs/blob/eeb2cff50119cfa7b724ac63c768d637ec01827a/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/machineLearningServices.json#L3361-L3372).